st2tests/st2tests/fixtures/generic/actions/action-with-output-schema.yaml

```diff
@@ -0,0 +1,19 @@
+---
+description: Action that executes an arbitrary Linux command on the localhost.
+enabled: true
+entry_point: ''
+name: local_schema
+pack: core
+parameters:
+  cmd:
+    required: true
+  sudo:
+    immutable: true
+    type: boolean
+runner_type: "local-shell-cmd"
+output_schema:
+  stdout:
```

Comment:
Since there's an `output_path` now, why do we need to put stdout here again?  Couldn't we simplify this to the following?

```
output_schema:
  type: object
  parameters:
    bob:
      type: string
```

Reply:
Correct, but the test that used this was removed. I'll fix this fixture when I add the test back in.